Neos.Flow/Tests/Functional/Reflection/Fixtures/DummyClassWithTypeHints.php

```diff
@@ -31,4 +31,8 @@ public function methodWithArrayTypeHint(array $array)
     public function methodWithArrayTypeHintAndAnnotation(array $array)
     {
     }
+
+    public function methodWithUnionReturnType(): string|false
```

Comment:
This in a test like that will probably break Testing for older PHP versions, wouldn't it? I guess we need some kind of language level detection to skip tests, which would break older PHP parsers - but I am not 100% sure.

The current Tasks only checked PHP >=8.0, if we were targetting 7.3 it would need to degrade gracefully in PHP 7.4

Reply:
(If we stay with 8.x as target, that probably is not neccessary - and I'm fine with all other aspects.)

Reply:
That is why i would prefer to target 8.0. Support 7.3 would require some additional adjustments.